src/main/java/org/folio/edge/sip2/repositories/ConfigurationRepository.java

```diff
@@ -45,11 +46,10 @@ public ConfigurationRepository(IResourceProvider<Object> resourceProvider) {
       ACSStatus acsStatus = null;
       if (acsConfiguration != null) {
         ACSStatus.ACSStatusBuilder builder = ACSStatus.builder();
-  
         builder.checkinOk(acsConfiguration.getBoolean("onlineStatus"));
         builder.acsRenewalPolicy(acsConfiguration.getBoolean("acsRenewalPolicy"));
         builder.checkoutOk(acsConfiguration.getBoolean("checkoutOk"));
-        builder.dateTimeSync(ZonedDateTime.now());
+        builder.dateTimeSync(ZonedDateTime.now(Clock.systemUTC()));
```

Comment:
You might want to consider passing in the `Clock` to the constructor. That way you can mock it in the unit tests or create a fixed time clock so you can test whether or not the date is as expected when returned. For reference, this is done in a couple of the other repos/unit tests.

Reply:
- Refactored it to pass in the clock to the constructor.

- Also created a Test utility class for frequently used clock/time formatting operations. Quite a few files have been changed because of this.